src/app/views/App.tsx

```diff
@@ -165,64 +172,86 @@ class App extends Component<IAppProps, IAppState> {
     });
   };
 
+  public toggleSidebar(): void {
+    const showSidebar = !this.state.showSidebar;
+    this.setState({ showSidebar });
+  }
+
+  public displayToggleButton = (mediaQueryList: any) => {
+    const showToggle = mediaQueryList.matches;
+    let showSidebar = true;
+    if (showToggle) {
+      showSidebar = false;
+    }
+    this.setState({ showToggle, showSidebar  });
+  }
+
   public render() {
     const classes = classNames(this.props);
     const { graphExplorerMode, error, actions, sampleQuery }: any = this.props;
+    const { showSidebar, showToggle } = this.state;
     const layout =
       graphExplorerMode === Mode.TryIt
-        ? 'col-sm-12'
-        : 'col-sm-12 col-lg-9';
+        ? 'col-sm-12 col-xs-12'
+        : 'col-xs-12 col-sm-12 col-lg-9 col-md-8';
     return (
       // @ts-ignore
       <ThemeContext.Provider value={this.props.appTheme}>
-      <FocusTrapZone>
-        <div className={`container-fluid ${classes.app}`}>
-          <div className='row'>
-            {graphExplorerMode === Mode.Complete && (
-              <div className={`col-sm-3 col-lg-3 col-md-3 ${classes.sidebar}`}>
-                <Sidebar />
-              </div>
-            )}
-            <div className={layout}>
-              {graphExplorerMode === Mode.Complete && <Authentication />}
-              {graphExplorerMode === Mode.TryIt && (
+        <FocusTrapZone>
+          <div className={`container-fluid ${classes.app}`}>
+            <div className='row'>
+              {graphExplorerMode === Mode.Complete && (
+                <div className={`col-sm-12 col-lg-3 col-md-4 ${classes.sidebar}`}>
+                  {showToggle && <IconButton
+                    iconProps={{ iconName: 'GlobalNavButton' }}
+                    className={classes.sidebarToggle}
+                    title='Remove sidebar'
+                    ariaLabel='Remove sidebar'
+                    onClick={() => this.toggleSidebar()}
+                  />}
+                  {showSidebar && <Sidebar />}
+                </div>
+              )}
+              <div className={layout}>
+                {graphExplorerMode === Mode.Complete && <Authentication />}
+                {graphExplorerMode === Mode.TryIt && (
```

Comment:
`graphExplorerMode` can either be `Complete or TryIt ` we don't need multiple checks. we can just use the complete check and return Authentication or the message bar.